tests/README.md

```diff
@@ -0,0 +1,10 @@
+# Test HCO
+
+Provide a simple container which contains `kubectl` and can retrieve and
+install HCO.
+
+## How to run it
+
+```
+docker run --rm -it -v $PWD/kubeconfig:/kubeconfig -e KUBECONFIG=/kubeconfig kubevirt/hco-tests:latest
```

Comment:
At some point we should publish the image to a registry.

Reply:
yep, also releasing manifests would be helpful on the ghithub release pages which are bound to a release.